### PR TITLE
Update incrementalCacheHandlerPath.mdx

### PR DESCRIPTION
**Pull request by jazsouf**
I was looking into `revalidateTag` intricacies when called from a server action vs route handler and found this little error in the return type of the function.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 90a9c1b3a3660b2f6860ccf3cb26d3075175d7d6

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ Returns `Promise<void>`.
 | --------- | -------- | ---------------------------- |
 | `tag`     | `string` | The cache tag to revalidate. |
 
-Returns `Promise<void>`. Learn more about [revalidating data](/docs/app/building-your-application/data-fetching/incremental-static-regeneration) or the [`revalidateTag()`](/docs/app/api-reference/functions/revalidateTag) function.
+`revalidateTag` does not return a value. Learn more about [revalidating data](/docs/app/building-your-application/data-fetching/incremental-static-regeneration) or the [`revalidateTag()`](/docs/app/api-reference/functions/revalidateTag) function.
```
Comment (ijjk) [inline docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx:50]: This does return a Promise per the type here https://github.com/vercel/next.js/blob/fc2519ea3d48393910caa6f49aaf90d6a70cacc1/packages/next/src/server/lib/incremental-cache/index.ts#L59-L61

Comment (ijjk) [reply docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx:50]: I believe the confusion is with the `revalidateTag` from `next/cache` which doesn't return a Promise

**Comment by ijjk** [comment]
Closing per-above

File: docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx (hunk 1 above)
Comment (jazsouf) [reply docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx:50]: Oh okey, my bad, thanks for clearing this up

**Pull request closed without merging**